README.md

```diff
@@ -1,13 +1,33 @@
 # Hangman Chat
 A simple chat with hangman where you can play hangman in with others.
 
+![hangman gif](kev_app.gif)
+
 # Table of Content
 - [Getting started](#getting-started)
-- [](#)
+- [Features](#features)
+- [What's used](#what's-used)
+- [To Do](#to-do)
 
 # Getting started
+Change the branch to hangman ;)
 Here is how to get started:
 1. Download or clone the repo `git clone https://github.com/kyunwang/hangman-chat.git`
 2. Run `npm install` or `yarn`
 3. Do a `npm start`
 4. Go to **localhost:3100** and you are ready to go.
+
+# Features
+* socket.io is beeing used to make the app realtime.  
+You can now use the app to  talk with eachother and send messages.  
```

Comment:
Another nitpick typo:
- double spaces between *to* and *talk*
- Missing space between *eachother*